libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp

```diff
@@ -682,6 +682,13 @@ uint32_t AP_Frsky_Telem::calc_batt(uint8_t instance)
 uint32_t AP_Frsky_Telem::calc_ap_status(void)
 {
     uint32_t ap_status;
+    
+    // prepare imu temp to a 6 bit value with a range from 20° - 83° value offset is 19
+    // 1 = 20°, 63 = 82°
```

Comment:
You may want to clarify the comments here, whereas, for the value stored on the 6 MSBs of ap_status: 0 (0x00) represents 19°C or less, and 63 (0x3f) represents 82°C or more (there is no value representing 83°C, or even 82°C).
1 (0x01) = 20°C, 62 (0x3e) = 81°C (63 not equal to 82°C, but instead means a value of 82°C or more).